.github/workflows/push-master.yml

```diff
@@ -0,0 +1,31 @@
+name: any-pr
```

Comment:
Same comment -> "Publish unstable release to NPM"

Reply:
With this proposal would you have the file name? publish-unstable-release-to-npm?

Reply:
its thats what the action is going, thats what I would name it.

I tend to prefer actions do 1 thing, and make them very generic so they can be copied easily without modification.